docs/site/content/en/docs/Tasks/machine-accounts/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Machine Accounts
+date: 2024-06-18
+weight: 1
+---
+
+A team manager can create `Machine Accounts` in his managed team(s). Those accounts can access Horreum with HTTP Basic authentication instead of authenticating with Keycloak. This is done on the `Machine Accounts` tab on the user profile (click on your name in upper right corner).
```

Comment:
```suggestion
A team manager can create `Machine Accounts` for their managed team(s). Those accounts can access Horreum with HTTP Basic authentication instead of authenticating with Keycloak. This is done on the `Machine Accounts` tab on the user profile (click on your name in upper right corner).
```